e2e/testcafe-devextreme/tests/dataGrid/keyboardNavigation/editOnKeyPress.screenshots.ts

```diff
@@ -62,3 +65,55 @@ const DATA_GRID_SELECTOR = '#container';
     await makeRowsViewTemplatesAsync(DATA_GRID_SELECTOR);
   });
 });
+
+test('Focused cell should not flick', async (t) => {
+  const dataGrid = new DataGrid(DATA_GRID_SELECTOR);
+  const firstCell = dataGrid.getDataCell(0, 0).element;
+  const secondCell = dataGrid.getDataCell(1, 0).element;
+  await t.click(firstCell);
+  await t.click(secondCell);
+
+  await ClientFunction(() => {
+    // @ts-expect-error
+    window.counter = 0;
+    (secondCell() as any as HTMLElement).addEventListener('focusin', () => {
+      // @ts-expect-error
+      window.counter += 1;
+    });
+  }, {
+    dependencies: { secondCell },
+  })();
+
+  await t.pressKey('M');
+  await t.pressKey('enter');
+
+  const focusEventCount = await ClientFunction(
+    // @ts-expect-error
+    () => window.counter,
+  )();
```

Comment:
Same comment for the inline anonymous ```ClientFunction```
Let's name this function and move on top of the test case code or move out it from this test case.

E.g:
```typescript
const getFocusInCount = () => ClientFunction(() => {
  const windowWithTestData = window as WindowWithTestData;
  return windowWithTestData.counter;
});
```

And in the test code:
```typescript
test('...', async (t) => {
  // other lines
  const focusEventCount = await getFocusInCount();
  // other lines
});
```